src/main/java/uk/gov/hmcts/reform/sscs/service/venue/VenueRpcDetails.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.reform.sscs.service.venue;
+
+import lombok.Getter;
+import uk.gov.hmcts.reform.sscs.model.VenueDetails;
+
+@Getter
+public class VenueRpcDetails {
+
+    private final VenueDetails venueDetails;
+    private final String rpc;
```

Comment:
Minor/Optional: can we change this to regionalProcessingCentre or is rpc well understand in this context?

Reply:
I think is part of the mapping, I can leave a comment with it